tests/10apidoc/00prepare.pl

```diff
@@ -42,3 +42,56 @@ sub new_User
 
    return $user;
 }
+
+
+# assert that the given members are in the body of a sync response
+sub assert_room_members {
+   my ( $body, $room_id, $member_ids ) = @_;
+
+   my $room = $body->{rooms}{join}{$room_id};
+   my $timeline = $room->{timeline}{events};
+
+   log_if_fail "Room", $room;
+
+   assert_json_keys( $room, qw( timeline state ephemeral ));
+
+   return assert_room_members_in_state( $room->{state}{events}, $member_ids );
+}
+
+
+# assert that the given members are present in a block of state events
+sub assert_room_members_in_state {
```

Comment:
somehow this name makes me think we're checking the state contains *at least* these members. Maybe `assert_state_room_members_matches` ?

Reply:
this also feels out of place. I'm not really keen on this file being used as a dumping ground for random assertion functions.

Maybe `10apidoc/09synced.pl` is a more appropriate home?